vendor/github.com/knative/test-infra/scripts/library.sh

```diff
@@ -368,7 +368,13 @@ function update_licenses() {
 function check_licenses() {
   # Fetch the google/licenseclassifier for its license db
   go get -u github.com/google/licenseclassifier
+  echo @@@@@
+  ls /home/prow/go/github.com/google/licenseclassifier
+  echo @@@@@
+  cat /home/prow/go/github.com/google/licenseclassifier/classifier.go
+  echo @@@@@
   # Check that we don't have any forbidden licenses in our images.
+  echo run_go_tool ./vendor/github.com/knative/test-infra/tools/dep-collector dep-collector -check $@
```

Comment:
I think ./vendor/github.com/knative/test-infra/tools/dep-collector is just missing because it was missing as import in pkg/dummy_test.go. Let me try to add this in the other PR and rerun.

Reply:
Sorry, misread the error log, it alteady breaks with running the licenseclassifier.

@adrcunha Please note that we have go modules enabled, so the licenseclassifier should be installed in the vendor dir, or ? (still fishing in the dark)